src/pages/MovieDetails.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
+import starIcon from '../assets/star.svg';
+
+const IMAGE_BASE_URL = "https://image.tmdb.org/t/p/";
+const POSTER_SIZE = "w342";
+const BACKDROP_SIZE = "w780";
+
+function MovieDetails() {
+  const [movie, setMovie] = useState({});
+  const { id } = useParams();
+
+  useEffect(() => {
+    const api_key = '917200cae8c9f306c0e6719af5d42134';
```

Comment:
So I just learned that in real-world projects, we shouldn't write the api key directly in the code. We should've used .env file instead, and git ignore the .env file to avoid pushing it to Github. But in this case, since our API key is free, it shouldn't be a big deal to have it in the code pushed to Github.

Reply:
Thank you! I saw that as well but didn’t found out how to solve it!

Reply:
Same here, we got information about the .env during the demo. I actually tried to do the setup with .env for our project after the demo but It didn't work and I think that's because github keeps track of files when pushed once already, or I just failed :)